### PR TITLE
[KIECLOUD-579] - update rhpam process-migration image with latest Qua…

### PR DESCRIPTION
**Pull request by spolti**
…rkus update
See: https://issues.redhat.com/browse/KIECLOUD-579

Signed-off-by: spolti <fspolti@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [x] Pull Request title is properly formatted: `[KIECLOUD-XYZ] Subject`, `[RHDM-XYZ] Subject` or `[RHPAM-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Attached commits represent units of work and are properly formatted
- [x] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [x] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

File: tests/features/rhpam/rhpam-process-migration.feature [hunk 1]
```diff
@@ -1,11 +1,61 @@
 @rhpam-7/rhpam-process-migration-rhel8
 Feature: RHPAM Process Migration tests
 
-  Scenario: Test REST API is available and valid
+  Scenario: verify if all labels are correctly set on kogito-builder image
```
Comment (jakubschwan) [inline tests/features/rhpam/rhpam-process-migration.feature:4]:
```suggestion
  Scenario: Verify if all labels are correctly set on rhpam-process-migration-rhel8 image
```

Comment (spolti) [reply tests/features/rhpam/rhpam-process-migration.feature:4]: nice catch LOL, copy and paste issue :)

**Review by desmax74 — APPROVED** [review]

File: tests/features/rhpam/rhpam-process-migration.feature (hunk 1 above)
Comment (desmax74) [reply tests/features/rhpam/rhpam-process-migration.feature:4]: Is still the old version, not the Jakub' suggestion

File: jboss-kie-process-migration/tests/bats/jboss-kie-process-migration.bats [hunk 2]
```diff
@@ -2,136 +2,95 @@
...
-  local expected
-
-  JBOSS_KIE_EXTRA_CLASSPATH="mysql-driver.jar"
-  expected="-Dthorntail.classpath=mysql-driver.jar"
-  configure_extra_classpath >&2
-  echo "JBOSS_KIE_EXTRA_CLASSPATH is ${JBOSS_KIE_EXTRA_CLASSPATH}" >&2
-  echo "Expected is ${expected}" >&2
-  [[ $JBOSS_KIE_EXTRA_CLASSPATH == "${expected}" ]]
+    touch ${JBOSS_HOME}/extra-classpath/mysql-driver.jar
+    JBOSS_KIE_EXTRA_CLASSPATH="${JBOSS_HOME}/extra-classpath/mysql-driver.jar"
```
Comment (desmax74) [inline jboss-kie-process-migration/tests/bats/jboss-kie-process-migration.bats:26]: Why is not included the PostgreSQL driver and is used only mysql ?

File: jboss-kie-process-migration/configure.sh [hunk 3]
```diff
@@ -16,10 +16,6 @@ cp -r ${ADDED_DIR}/launch/* ${LAUNCH_DIR}
 mkdir -p ${CONFIG_DIR}
 cp -r ${ADDED_DIR}/configuration/* ${CONFIG_DIR}
 
-mkdir -p ${JDBC_DRIVERS}
-link /usr/lib/java/mariadb-java-client.jar ${JDBC_DRIVERS}/mariadb-java-client.jar
-link /usr/share/java/postgresql-jdbc/postgresql.jar ${JDBC_DRIVERS}/postgresql-jdbc.jar
```
Comment (desmax74) [inline jboss-kie-process-migration/configure.sh:21]: Here was included Mysql/Mariadb and Postgresql drivers , with the changes PostgreSQL drivers aren't included is it correct ?

Comment (spolti) [reply jboss-kie-process-migration/configure.sh:21]: yes, jdbc drivers are now Quarkus extensions, see: https://quarkus.io/guides/datasource

File: jboss-kie-process-migration/tests/bats/jboss-kie-process-migration.bats (hunk 2 above)
Comment (spolti) [reply jboss-kie-process-migration/tests/bats/jboss-kie-process-migration.bats:26]: it is just a fictitious name, doesn't really matter.

File: tests/features/rhpam/rhpam-process-migration.feature (hunk 1 above)
Comment (spolti) [reply tests/features/rhpam/rhpam-process-migration.feature]: updated, thanks.

**Review by msmagnanijr — APPROVED** [review]
lgtm

**Pull request merged**